dotnet-datetime/CheckIfDateIsLessThanOrEqualToToday/CheckIfDateIsLessThanOrEqualToToday/CheckDateMethods.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Globalization;
+
+namespace CheckIfDateIsLessThanOrEqualToToday;
```

Comment:
Hi @jshergal, the `DateOnly` struct does not have a definition for `Today`, as discussed in this issue: https://github.com/dotnet/runtime/issues/53498.

So, since we are dealing with today's date, instead of having something as clear as this: `DateTime.Today`, we would have to replace it with `DateOnly.FromDateTime(DateTime.Today)`. Should we go ahead with this?

Reply:
@cjaynjoku yes, you create a DateOnly from a DateTime, you can use DateTime.Now, UtcNow, Today, etc.

The interesting part in the discussion with the DotNet link you shared is the comments there regarding the fact that DateOnly should not care about the timezone, so that may actually be a greater motivation to focus on DateOnly rather than DateTime, and since in the code we are creating a date from a string, we can then create today's date using `DateOnly.FromDateTime(DateTime.Today)`, and use it without having to think about kinds and timezones at all.

Reply:
Okay @jshergal, let's focus on just the DateOnly struct.